ansible_roles/roles/aws_create/files/tf/disks_gp3.tf

```
@@ -9,7 +9,7 @@ resource "aws_ebs_volume" "disk_volume" {
   }
 }
 
-resource "aws_volume_attachment" "disk_attachement" {
+resource "aws_volume_attachment" "disk_attachemnt" {
```

```suggestion
resource "aws_volume_attachment" "disk_attachment" {
```

Verify the terraform stuff isn't referenced anywhere else

---

Have done so.